sdk/src/main/java/com/mapbox/maps/MapboxMap.kt

```diff
@@ -1691,7 +1691,7 @@ class MapboxMap :
   @MapboxExperimental
   override fun setMapProjection(mapProjection: MapProjection) {
     checkNativeMap("setMapProjection")
-    val expected = nativeMap.setMapProjection(mapProjection.toValue())
+    val expected = nativeMap.setStyleProjection(Value.nullValue())
```

Comment:
ok with null here as this will be revisited later

Reply:
`null` value means it will always return `Mercator`, but we can discuss what we need to do about these methods in comment https://github.com/mapbox/mapbox-maps-android/pull/1244#discussion_r840677196